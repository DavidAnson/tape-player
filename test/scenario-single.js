// @ts-check

"use strict";

const tape = require("tape");
require("../index.js");

tape("single test", (test) => {
  test.plan(1);
  test.ok(true);
});
