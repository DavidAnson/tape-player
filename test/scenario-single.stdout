single test

Tests:    1
Asserts:  1
Failures: 0
